frontend/src/features/approval/Approval.tsx

```diff
@@ -2,7 +2,7 @@ import React, { ReactElement, useCallback, useContext, useEffect, useState } fro
 import { Button, ButtonProps, Spinner } from '@chakra-ui/react'
 import { checkApproval, sendApproval } from './approvalAPI';
 import { ProviderContext, ERC20Context, CurrentAddressContext, YieldTokenCompoundingContext, SymfoniContext } from '../../hardhat/SymfoniContext';
-import { BigNumber, ContractReceipt, providers } from 'ethers';
+import { BigNumber, ContractReceipt, ethers, providers } from 'ethers';
```

Comment:
is there a reason we are importing the whole module and yet also importing section classes?